src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -370,6 +434,90 @@ TxQ::eraseAndAdvance(TxQ::FeeMultiSet::const_iterator_type candidateIter)
 
 }
 
+auto
+TxQ::erase(TxQ::TxQAccount& txQAccount,
+    TxQ::TxQAccount::TxMap::const_iterator begin,
+        TxQ::TxQAccount::TxMap::const_iterator end)
+            -> TxQAccount::TxMap::iterator
+{
+    for (auto it = begin; it != end; ++it)
+    {
+        byFee_.erase(byFee_.iterator_to(it->second));
```

Comment:
I was double-checking the docs when you followed-up yourself. IIRC, though, `iterator_to` is nearly-free, since it's mostly just a fancy pointer cast.